cypress/integration/pattern/pattern.test.js

```diff
@@ -0,0 +1,174 @@
+import 'cypress-file-upload';
+import { config } from '../../../src/configs';
+import {
+  getAllPatterns,
+  typeValue,
+  addPattern,
+  addPatternError,
+  updateValue,
+  updatePattern,
+  deletePattern
+} from './pattern.variables';
+import statuses from '../../../src/configs/statuses';
+import routes from '../../../src/configs/routes';
+import { patternTranslations } from '../../../src/translations/pattern.translations';
+
+describe('pattern tests', () => {
+  beforeEach(() => {
+    cy.login(Cypress.env('ADMIN_LOGIN'), Cypress.env('ADMIN_PASSWORD'));
+    cy.stubRequest('getAllPatterns', getAllPatterns).as('getAllPatterns');
+    cy.visit('/');
```

Comment:
same

Reply:
We don't have home page on '/' route. When we merge pr with homepage I will fix it